Examples/Scripts/Python/full_chain_itk.py

```diff
@@ -61,7 +61,11 @@
     field,
     rnd=rnd,
     preSelectParticles=ParticleSelectorConfig(
-        eta=(-4.0, 4.0), pt=(150 * u.MeV, None), removeNeutral=True
+        rho=(0.0 * u.mm, 28.0 * u.mm),
+        absZ=(0.0 * u.mm, 1.0 * u.m),
```

Comment:
can you clarify this cut in Z? We are removing particles originating from `(0,0,0)`?

Reply:
ah ok, I have to read it the other way... we are accepting only particles originating in -1 m < z < 1m

Reply:
this will remove everything outside of this range. so we are selecting `-1m <= z <= 1m` and `0mm <= r <= 28mm`. not sure about the `<` vs `<=` but I can double check.

the problem is that pythia will create particles inside the detector which will make Fatras fail from time to time

Reply:
so currently within means the following

```
  auto within = [](double x, double min, double max) {
    return (min <= x) and (x < max);
  };
  ```

Reply:
I would expect the event simulation to produce the secondaries and the event generation should stick to the beampipe. because pythia does not know about the detector and the magnetic field. but maybe I do not have the whole picture

Reply:
Yes, I think you are right. I just don't know what should happen to the excluded evgen particles outside the beampipe. If they are just spurious pileup collisions, then they should be killed with extreme prejudice as unphysical. But if they are decays, then we'd be cutting off the longer decays and they would interact badly with the simulation decay.

Sorry, I don't know how this works, so I probably just worry needlessly.